Resources/Private/Templates/Event/ICalendar.txt

```diff
@@ -6,15 +6,15 @@ METHOD:PUBLISH
 BEGIN:VEVENT
 UID:{event.uid}-{event.pid}@{typo3Host}
 CLASS: PUBLIC
-SUMMARY:{event.title -> f:format.htmlentitiesDecode()}
+SUMMARY:<e:format.ICalendarDescription>{event.title -> f:format.htmlentitiesDecode()}</e:format.ICalendarDescription>
```

Comment:
With #992 a new argument has been introduced to the ICalendarDescriptionViewHelper. This must be used here to ensure, that the first line will. not exceed 75 chars. 

For `SUMMARY:` this is `<e:format.ICalendarDescription substractChars="8">`

Reply:
Done.